packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -169,6 +174,8 @@ class DropdownButton extends React.Component {
       theme.className,
     );
 
+    const buttonAriaLabel = (inputAttrs && inputAttrs['aria-label']) ? inputAttrs['aria-label'] : '';
```

Comment:
these can be achieved even without `inputAttrs` with `customProps`. so when ever consumer pass aria-label we can do 
`const buttonAriaLabel = (customProps['aria-label']) ? customProps['aria-label'] : '';`

but the actual reason to introduce attrs object is to support consumer passing attributes to button.  In this case we need add new `attrs` prop to button element by removing aria-label from it since button has already got aria-label.
```js
 let buttonAriaLabel;
 if (inputAttrs && inputAttrs['aria-label']) {
      buttonAriaLabel = inputAttrs['aria-label'];
      delete inputAttrs['aria-label'];
 }

<button
{...inputAttrs}
```